Python/Product/Cookiecutter/View/CookiecutterSearchPage.xaml.cs

```diff
@@ -100,7 +100,7 @@ internal partial class CookiecutterSearchPage : Page {
 
         private void TreeView_SelectedItemChanged(object sender, RoutedPropertyChangedEventArgs<object> e) {
             var val = e.NewValue as TemplateViewModel;
-            ViewModel.SelectTemplate(val).DoNotWait();
+            ViewModel.SelectTemplateAsync(val).DoNotWait();
 
             SelectedTemplateChanged?.Invoke(this, EventArgs.Empty);
```

Comment:
Should we really be raising the event after the call above has completed? It's actually fairly easy - just make this event handler `async void` and await the SelectTemplateAsync call (event handlers are basically why `async void` was added).